pyuvdata/tests/test_uvflag.py

```diff
@@ -1300,6 +1300,24 @@ def test_to_baseline_force_pol():
     assert uvf.flag_array.mean() == ntrue / uvf.flag_array.size
 
 
+def test_to_baseline_force_pol_Npol_gt_1():
+    uv = UVData()
+    uv.read_miriad(test_d_file)
+    uvf = UVFlag(uv)
+    uvf.to_waterfall()
+    uvf.to_flag()
+    uvf.flag_array[0, 10, 0] = True  # Flag time0, chan10
+    uvf.flag_array[1, 15, 0] = True  # Flag time1, chan15
+
+    uv2 = copy.deepcopy(uv)
+    uv2.polarization_array[0] = -6
+    uv += uv2
+    uvf.to_baseline(uv, force_pol=True)
+    assert np.all(uvf.baseline_array == uv.baseline_array)
+    assert np.all(uvf.time_array == uv.time_array)
+    assert np.array_equal(uvf.polarization_array, uv.polarization_array)
```

Comment:
It looks like the bug that was fixed was that `Npol` wasn't updated. Could you add a check for it here so you're testing the bug?

Reply:
Sure thing. I did end up covering that b/c the failure would appear any time a `uvf.check()` was called, which I have covered, but I can just add an explicit check